### ISSUE TITLE
Segmentation training compared yolov5 - yolov8

### ISSUE THREAD

**Issue by wvalcke** [ultralytics/ultralytics#957]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I've trained a yolov5 model which consistently gives a map 0.5:0.95 of 99.8 % on boxes, 99.9% on segments. (small model)
Using the new yolov8, and using same dataset it hardly gets a map 0.5:0.95 of 96-97%, the segment loss gives better results up till 98, 99% (small model)
I do not understand why those results are so different, i use the same hyperparameters (mosaic is active) , ....
If anyone has seen this also, maybe just share some ideas, or confirm you see the same behavior.
Another difference, i can train for 1200 epochs on the yolov5 model, giving at the end very smooth loss curves, but doing the same on yolov8 gives at around epoch 400-500 a maximum for map (lowest loss), then the box/dfl loss increases for the validation set and gets worse the more epochs pass. The cls loss and segment loss do not increase.

If somebody has some tips/ideas to try out, please let me know.

Thanks!

**Comment by wvalcke** [comment]
Hi @JustasBart 
I'll close this question as it it definitely being caused by issue #1010.
Beware, it's really important as it can really influence your training results.

**Comment by JustasBart** [comment]
Thanks for investigating this matter @wvalcke! This seems rather important to get right.
